src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/WebhookConfiguration.java

```diff
@@ -67,16 +66,22 @@
             HookEventType.SERVER_PULL_REQUEST_MERGED.getKey(),
             HookEventType.SERVER_PULL_REQUEST_DECLINED.getKey(),
             HookEventType.SERVER_PULL_REQUEST_DELETED.getKey(),
+            // only on v5.10 and above
             HookEventType.SERVER_PULL_REQUEST_MODIFIED.getKey(),
             HookEventType.SERVER_PULL_REQUEST_REVIEWER_UPDATED.getKey(),
             // only on v7.x and above
             HookEventType.SERVER_PULL_REQUEST_FROM_REF_UPDATED.getKey()
     ));
 
     /**
-     * The list of events available in Bitbucket Server v6.x.
+     * The list of events available in Bitbucket Server v5.10+.
      */
-    private static final List<String> NATIVE_SERVER_EVENTS_v6 = Collections.unmodifiableList(NATIVE_SERVER_EVENTS_v7.subList(0, 6));
+    private static final List<String> NATIVE_SERVER_EVENTS_v5_10 = Collections.unmodifiableList(NATIVE_SERVER_EVENTS_v7.subList(0, 7));
```

Comment:
```suggestion
    private static final List<String> NATIVE_SERVER_EVENTS_v6 = Collections.unmodifiableList(NATIVE_SERVER_EVENTS_v7.subList(0, 7));
```

Reply:
Do you think this could be confusing since these also apply to v5.10?

Reply:
I'd rather have a change the name of the variable. I expect v6 to be supported for longer, at some point we'll want to remove support for 5.x and I would like to make that process be a matter of deleting lines that refer to 5.x (not changing variable names again.

I'd also be okay with: 
```java
    /**
     * The list of events available in Bitbucket Server v5.10 to 5.16 is the same a Bitbucket Server 6.x.        
     */
    private static final List<String> NATIVE_SERVER_EVENTS_v5_10 = NATIVE_SERVER_EVENTS_v6;
```